src/collision.cpp

```diff
@@ -25,11 +25,13 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "gamedef.h"
 #ifndef SERVER
 #include "client/clientenvironment.h"
+#include "client/localplayer.h"
 #endif
 #include "serverenvironment.h"
 #include "serverobject.h"
 #include "util/timetaker.h"
 #include "profiler.h"
+#include <iomanip>
```

Comment:
what is this needed for?

Reply:
Probably leftover from my testing, I must have been outputting some text, gonna check and get rid of.